### PR TITLE
Issue #1511 - Renaming jetty-all to jetty-tutorial

### PR DESCRIPTION
**Pull request by joakime**
Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

### CODE REVIEW

File: jetty-home/pom.xml [hunk 1]
```diff
@@ -145,7 +145,7 @@
                 org.eclipse.jetty.orbit,org.eclipse.jetty.http2,org.eclipse.jetty.websocket,org.eclipse.jetty.fcgi,org.eclipse.jetty.toolchain,org.apache.taglibs
               </excludeGroupIds>
               <excludeArtifactIds>
-                jetty-all,apache-jsp,apache-jstl,jetty-start,jetty-spring
+                jetty-all,jetty-tutorial,apache-jsp,apache-jstl,jetty-start,jetty-spring
```
Comment (sbordet) [inline jetty-home/pom.xml]: Is there a reason for `jetty-all` to still be there?

File: jetty-websocket/README.TXT [hunk 2]
```diff
@@ -15,10 +15,10 @@ A TestServer and TestClient class are available, and can be run either directly
 imported), or from the command line with
 
 
-  java -cp jetty-aggregate/jetty-all/target/jetty-all-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
+  java -cp jetty-aggregate/jetty-tutorial/target/jetty-tutorial-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
```
Comment (sbordet) [inline jetty-websocket/README.TXT]: Should not be this `jetty-tutorial-10.x.y.jar`? And the servlet-api jar changed accordingly?

File: jetty-websocket/README.TXT [hunk 3]
```diff
@@ -15,10 +15,10 @@ A TestServer and TestClient class are available, and can be run either directly
...
   org.eclipse.jetty.websocket.TestServer  --help 
 
-  java -cp jetty-aggregate/jetty-all/target/jetty-all-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
+  java -cp jetty-aggregate/jetty-tutorial/target/jetty-tutorial-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
```
Comment (sbordet) [inline jetty-websocket/README.TXT]: Ditto.

File: jetty-home/pom.xml (hunk 1 above)
Comment (joakime) [reply jetty-home/pom.xml]: I can't think of one. 
I'll remove it.

File: jetty-websocket/README.TXT (hunk 2 above)
Comment (joakime) [reply jetty-websocket/README.TXT]: yow, old doc.  I'll have to update it too.

**Comment by gregw** [comment]
As I said in the issue, I'm not OK with this change.
If this artifact is no longer fit for use, then let's deprecate it.
`jetty-tutorial` is not a meaningful name to me and I don't think it well covers all the use-cases for a single jar.

**Comment by joakime** [comment]
This has a -1 review.  Closing without merge.
(Still have to fix that old README in websocket though)

**Pull request closed without merging**